.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Setup Deno
         run: |
-          curl -fsSL https://deno.land/x/install/install.sh | sh ${{ matrix.deno == 'old' && '-s v1.9.1' || '' }}
+          curl -fsSL https://deno.land/x/install/install.sh | sh ${{ matrix.deno == 'old' && '-s v1.11.0' || '' }}
```

Comment:
It seems 1.11.0 is the oldest deno version which can pass the CI with the latest deno_std